src/browser/modules/Frame/styled.tsx

```diff
@@ -122,8 +123,16 @@ export const StyledFrameContents = styled.div<FullscreenProps>`
 
 export const StyledFrameStatusbar = styled.div<FullscreenProps>`
```

Comment:
This is the bottom panel used in CypherFrame, when we're **not** showing a graph visualization